connector/servicegraphconnector/README.md

```diff
@@ -0,0 +1,143 @@
+# Service Graph Connector
+
+| Status                   |                                                           |
+|------------------------- |---------------------------------------------------------- |
+| Stability                | [in development]                                          |
+| Supported pipeline types | See [Supported Pipeline Types](#supported-pipeline-types) |
+| Distributions            | []                                                        |
+
+The `count` connector can be used to count spans, data points, or log records.
```

Comment:
It mentions the `count` connector, we can probably remove it.

Reply:
Whoops, good catch.

Reply:
Removed